### ISSUE TITLE
Translation check

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Capabilities#507]
Can we make sure of two things for translations of this plugin?

- Free and Pro are using different text domains
- Free and Pro have separate translation files

**Comment by olatechpro** [comment]
Yes @stevejburge Capabilities Free and Pro has different text domain and different translation files.
